frontend/lib/src/components/elements/BokehChart/BokehChart.tsx

```diff
@@ -79,6 +79,9 @@ export function BokehChart({
 
   const updateChart = (data: any): void => {
     const chart = document.getElementById(chartId)
+    if (chart) {
+      chart.setAttribute("data-testid", "stBokehChart")
+    }
```

Comment:
Will leave to your discretion, but should this `data-testid` be part of the return statement later in this file or is it better set as attribute on chart like above?
```
return <div id={chartId} className="stBokehChart" data-testid="stBokehChart" />
```

Reply:
Hmmm that's a good point. I think we can probably just add it to the div so it's more consistent with other files.